.travis.yml

```diff
@@ -129,9 +129,10 @@ script:
     # compile / install Julia
     - make $BUILDOPTS NO_GIT=1 prefix=/tmp/julia install | moreutils/ts -s "%.s"
     - if [ `uname` = "Darwin" ]; then
-        for name in suitesparseconfig spqr umfpack colamd cholmod amd suitesparse_wrapper; do
-            install -pm755 usr/lib/lib${name}*.dylib* /tmp/julia/lib/julia/;
+        for name in suitesparseconfig spqr umfpack colamd cholmod amd; do
+            install -pm755 usr/lib/julia/lib${name}*.dylib* /tmp/julia/lib/julia/;
```

Comment:
I vaguely thought that we already copy everything inside this folder already, but doesn't hurt anything to do it again here in any case

Reply:
Yes, this also looked suspicious to me, but since I can't test this locally I figured I'd better leave it as-is. I can remove it if you think it's redundant.

Reply:
looking at the travis logs, it seems to confirm my expectation. There's doesn't seem to be any harm in leaving this in the .travis file either though, so do whatever you think best.
```
1393.707195 for suffix in libamd libcamd libccolamd libcholmod libcolamd libumfpack libspqr libsuitesparseconfig libpcre2-8 libgmp libmpfr libarpack libLLVM libopenblas ; do \

1393.707266 		lib=/Users/travis/build/JuliaLang/julia/usr/lib/julia/${suffix}.dylib; \

1393.707286 		/Users/travis/build/JuliaLang/julia/contrib/install.sh 755 $lib /tmp/julia/lib/julia ; \

1393.707304 	done
```

Reply:
It worked!